netbeans-jakartaee-war-archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" 
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>${groupId}</groupId>
+    <artifactId>${artifactId}</artifactId>
+    <version>${version}</version>
+    <packaging>war</packaging>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>${javaVersion}</maven.compiler.source>
+        <maven.compiler.target>${javaVersion}</maven.compiler.target>
```

Comment:
would it be possible to make it use `maven.compiler.release` for versions > 8?

Reply:
I implemented a prototype where this is adjusted in the post project creation groovy script:  https://github.com/asbachb/netbeans-mavenutils-archetypes/blob/master/netbeans-jakartaee-war-archetype/src/main/resources/META-INF/archetype-post-generate.groovy#L10-L21

Not 100% if I like it, but it works.

Reply:
this is pretty cool. I somehow forgot about the fact that maven can run groovy scripts as project creation step.

This will likely also allow us to set the right JVM flags for netbeans modules:
https://github.com/apache/netbeans/tree/master/nbbuild/jms-config

once we update this archetype. Thanks!